### PR TITLE
Document `gp ports list` command

### LINKED ISSUE #2171: Document `gp ports list`
The `gp ports list` (https://github.com/gitpod-io/gitpod/pull/10388) is not yet documented, we need to update: 

- https://www.gitpod.io/docs/config-ports
- https://www.gitpod.io/docs/command-line-interface

### PR DESCRIPTION
**Pull request by felladrin**
## Description

Document the `gp ports list` command.

## Related Issue(s)

Resolves #2171

## How to test

Confirm if the following pages* mention the `gp ports list` command:
- [docs/config-ports](https://deploy-preview-2200--gitpod-kumquat.netlify.app/docs/config-ports)
- [docs/command-line-interface](https://deploy-preview-2200--gitpod-kumquat.netlify.app/docs/command-line-interface)

_*These pages are from the preview environment of this PR._

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2200"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2200"></a>

### CODE REVIEW

**Review by loujaybee — APPROVED** [review]

File: gitpod/docs/command-line-interface.md [hunk 1]
```diff
@@ -148,3 +149,13 @@ Alternatively, specify the `Terminal ID` that you can see with `gp tasks list`:
 ```sh
 gp tasks attach <id>
 ```
+
+## ports
```
[the remark this answers is not in the document]
Comment (felladrin) [reply gitpod/docs/command-line-interface.md:153]: 💡  Let me update it.

I'll take the chance to update "Programatically" to "Program**m**atically" on the paragraph from the screenshot above.

[the remark this answers is not in the document]
Comment (felladrin) [reply gitpod/docs/command-line-interface.md:153]: Updated.
I couldn't think of anything better than this. Let me know if you have a better idea.

[the remark this answers is not in the document]
Comment (loujaybee) [reply gitpod/docs/command-line-interface.md:153]: Added a small suggestion, but looks good! Thanks 🚀

File: gitpod/docs/command-line-interface.md [hunk 2]
```diff
@@ -148,3 +149,15 @@ Alternatively, specify the `Terminal ID` that you can see with `gp tasks list`:
...
+
+Provides a way to manage the ports defined in [.gitpod.yml](/docs/references/gitpod-yml) plus any other opened during the workspace lifetime.
```
[the remark this answers is not in the document]
Comment (felladrin) [reply gitpod/docs/command-line-interface.md:155]: Thanks!

Just to confirm: "a way to manage a workspace's ports" or "a way to manage workspaces ports"?

**Pull request merged**